### ISSUE TITLE
Pausing debugging uses incorrect path to `kill`

### ISSUE THREAD

**Issue by expipiplus1** [microsoft/vscode-cpptools#10124]
### Environment

- OS and version: Linux (NixOS unstable)
- VS Code: 1.71.2
- C/C++ extension: vscode-cpptools
- GDB / LLDB version: gdb 12.1

### Bug Summary and Steps to Reproduce

Bug Summary:

On a machine with `kill` in `$PATH` but not in `/bin/`:

- Start debugging a long-running program using gdb
- Press the "pause" button to pause execution
    
    ![image](https://user-images.githubusercontent.com/857308/200511203-62efc122-eddd-48b2-85c6-ed9837664d71.png)

- Observe that something somewhere is trying to use `/bin/kill` rather than finding `kill` in `$PATH`

    ```
    Stopping due to fatal error: Win32Exception: An error occurred trying to start process '/bin/kill' with working directory '/home/e/my-project'. No such file or directory
    ```

### Debugger Configurations

```shell
{
    "version": "0.2.0",
    "configurations": [
      {
        "name": "C/C++: debug",
        "type": "cppdbg",
        "request": "launch",
        "program": "a.out",
        "args": [],
        "stopAtEntry": false,
        "cwd": "${workspaceFolder}",
        "environment": [],
        "externalConsole": false,
        "MIMode": "gdb",
        "miDebuggerPath": "gdb",
        "logging": { "engineLogging": true },
        "setupCommands": [
          {
            "description": "Enable pretty-printing for gdb",
            "text": "-enable-pretty-printing",
            "ignoreFailures": true
          }
        ],
        "preLaunchTask": "make debug"
      }
    ]
  }
```

### Debugger Logs

```shell
...
1: (857) <--exec-continue
--> E (output): {"type":"event","event":"output","body":{"category":"console","output":"1: (857) ->^running\n"},"seq":481}
1: (857) ->^running
--> E (output): {"type":"event","event":"output","body":{"category":"console","output":"1: (857) ->*running,thread-id=\"all\"\n"},"seq":483}
1: (857) ->*running,thread-id="all"
--> E (output): {"type":"event","event":"output","body":{"category":"console","output":"1: (857) ->(gdb)\n"},"seq":485}
1: (857) ->(gdb)
<--   C (pause-12): {"command":"pause","arguments":{"threadId":68743},"type":"request","seq":12}
Stopping due to fatal error: Win32Exception: An error occurred trying to start process '/bin/kill' with working directory '/home/e/my-project'. No such file or directory
```
```


### Other Extensions

_No response_

### Additional Information

_No response_

**Comment by WardenGnaw** [comment]
Thank you for reporting this issue.

Notes:
This code can be found at https://github.com/microsoft/MIEngine/blob/fd190c03bab7f332e522aa2bfb2f8b755206c787/src/MICore/UnixUtilities.cs#L262

**Comment by sean-mcmanus** [comment]
Your fix is available with 1.13.5 (pre-release, release candidate): https://github.com/microsoft/vscode-cpptools/releases/tag/v1.13.5
